content/pages/help.md

```diff
@@ -602,7 +602,7 @@ download page for a specific build, or use git to get an up to date copy of
 the source, see [here](https://wiki.freenetproject.org/Development) for details. Further
 instructions for building and deploying the server are included with the code
 itself. Generally speaking, joining our IRC channel is a good idea: [#freenet
-on chat.freenode.net](irc://chat.freenode.net/freenet)
+on libera.chat](rcs://irc.libera.chat:6697/freenet)
```

Comment:
Looks like `rcs` should be `ircs`?

Reply:
arg, yes!

Reply:
See https://github.com/freenet/website/commit/e4caa1cfd604a11d9c2218f72ad0b88f2bb2f5fb